rerun_py/rerun_bindings/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, TypeAlias, Union
```

Comment:
Why add these imports here?

I removed them in https://github.com/rerun-io/rerun/pull/7794